pkg/ccl/changefeedccl/sink_sql.go

```diff
@@ -106,7 +109,7 @@ func makeSQLSink(
 		topics:      topics,
 		hasher:      fnv.New32a(),
 		targetNames: targetNames,
-		metrics:     m,
+		metrics:     mb(false),
```

Comment:
wouldn't that depend where the sql traffic going?

Reply:
Yes, but adding a reliable way of knowing where the sql traffic was going seemed like it deserves its own thought so for now I think we should error in favor of the user.  I'll add a comment.